agent/provider/Cargo.toml

```diff
@@ -25,7 +25,8 @@ ya-utils-cli = "0.1"
 ya-utils-path = "0.1"
 ya-utils-process = { version = "0.2", features = ['lock'] }
 ya-std-utils = "0.1"
-golem-certificate = { git = "https://github.com/golemfactory/golem-certificate", rev = "50c98b6286b00ddc98ddd46157a26a03cfe0beec" }
+golem-certificate = { git = "https://github.com/golemfactory/golem-certificate", rev = "d12fc54a958e3ef04470b66127b5435c30a7222f" }
+# golem-certificate = { path = "../../../golem-certificate" }
```

Comment:
I think this is accidentally left in for local development

Reply:
Left it on purpose because we have(had) few such lines in Cargos. Useful to easier switch when working on library. Removed it.